_posts/2015-06-18-our-definition-of-done.markdown

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title:  "Our Definition of \"Done\""
+date:   2015-06-18 11:37:00
+categories: ["software development", "software quality"]
+author: etagwerker
+---
+
+Quality should be present in everything you do, but it should be balanced with the time you spend working on a feature. Does it feel like you've been working on this feature for a **really long time**? Maybe you have. Is it "done"?
+
+That is a tough question, so I'll write down _our definition of done_.
+
+## When is a feature done?
+
+* The feature does what it is expected
```

Comment:
@etagwerker I'd change this line to be either: `The feature does what it is expected to do` or `The feature does what is expected of it`. It feels like it's missing something to me.